packages/manager/modules/pci/src/projects/project/ai/apps/components/delete/translations/Messages_fr_FR.json

```diff
@@ -2,7 +2,7 @@
   "pci_ai_apps_delete_success": "Votre app a été supprimée avec succès",
   "pci_ai_apps_delete_error": "Une erreur est survenue durant la suppression de l'app : {{ message }}",
   "pci_ai_apps_delete": "Suppression de l'app",
-  "pci_ai_apps_delete_common_delete": "Supprimer",
+  "pci_ai_apps_delete_common_delete": "Confirmer",
   "pci_ai_apps_delete_common_cancel": "Annuler",
-  "pci_ai_apps_delete_warning": "Êtes-vous sûr de vouloir supprimer l'app {{ appId }} ?"
+  "pci_ai_apps_delete_warning": "Êtes-vous sûr de vouloir supprimer l'app <strong>{{ appId }}</strong> ?"
```

Comment:
We must avoid html in translation files. Instead you can have the `strong` tag in the template like this:

`data-translate-values="{ appId: '<strong>' + $ctrl.app.name + '</strong>' }"`